python/python/tests/test_lance.py

```diff
@@ -163,19 +163,7 @@ def test_create_index_shuffle_params(tmp_path):
         shuffle_partition_concurrency=10,
     )
 
-    embs = dataset.to_table()["emb"].to_numpy(zero_copy_only=False)
-
-    for emb in embs:
-        distance = dataset.to_table(
-            nearest={
-                "column": "emb",
-                "q": emb,
-                "k": 1,
-                "nprobes": 1,
-                "refine_factor": 5,
-            }
-        )["_distance"].to_pylist()[0]
-        assert abs(distance) < 1e-6
+    sanity_check_vector_index(dataset, "emb")
```

Comment:
Lets not use production code as assert in test?

If the production code is buggy, this test becomes invalid.